src/software/world/ball.h

```diff
@@ -6,6 +6,7 @@
 
 #include "software/geom/point.h"
 #include "software/util/time/timestamp.h"
+#include "software/world/ball_state.h"
 
 class Ball final
```

Comment:
If this class maintains an invariant that there is at least one `BallState` in the circular buffer, we should make a javadoc comment for this class and indicate as such.

Reply:
done

Reply:
Did we add a javadoc here? I don't see one if so?

Reply:
I added it on top of the states_ buffer in ball_state.h